botocore/httpsession.py

```diff
@@ -201,6 +223,7 @@ def _get_proxy_manager(self, proxy_url):
             proxy_headers = self._proxy_config.proxy_headers_for(proxy_url)
             proxy_manager_kwargs = self._get_pool_manager_kwargs(
                 proxy_headers=proxy_headers)
+            proxy_manager_kwargs.update(**self._proxies_kwargs)
```

Comment:
Does this get ignored if they're using an older version of `urllib3`? Do we want to be more opinionated on what happens in that case?

Reply:
They're absorbed into ` **connection_pool_kw` on older versions of urllib3 and ignored. I tested against the versions we support and things worked as expected.